include/internal/ktls.h

```diff
@@ -67,11 +68,20 @@ static ossl_inline int ktls_send_ctrl_message(int fd, unsigned char record_type,
     return -1;
 }
 
+
```

Comment:
could you please remove this empty line.

Reply:
Done